doc/modules/cassandra/pages/new/messaging.adoc

```diff
@@ -86,7 +86,7 @@ on-wire size of the message being sent or received.
 . The basic per-link limit is consumed in isolation before any endpoint
 or global limit is imposed. Each node-pair has three links: urgent,
 small and large. So any given node may have a maximum of
-`N*3 * (internode_application_send_queue_capacity_in_bytes + internode_application_receive_queue_capacity_in_bytes)`
+`N*3 * (internode_application_send_queue_capacity in bytes + internode_application_receive_queue_capacity in bytes)`
 messages queued without any coordination between them although in
```

Comment:
Instead of...

> So any given node may have a maximum of...

...what if we try...

> Any given node may have a maximum of `N*3 * (internode_application_send_queue_capacity + internode_application_receive_queue_capacity)` of message data queued without any coordination between them...

Reply:
Not related to these docs, but I see the benefit so I will change it :-)

Reply:
We're being such good citizens ;)